src/OnboardingSPA/components/App/index.js

```diff
@@ -223,12 +226,57 @@ const App = () => {
 		}
 	}
 
+	const handlePreviousStepTracking = () => {
+		const previousStep = window.nfdOnboarding?.previousStepID;
+		if ( typeof previousStep !== 'string' ) {
+			window.nfdOnboarding.previousStepID = location.pathname;
+			HiiveAnalytics.dispatchEvents();
+			return;
+		}
+
+		if ( previousStep.includes( 'products' ) ) {
+			trackHiiveEvent( 'products-info', {
+				productCount:
```

Comment:
Instead of just saving the data that might be in flow, should we be listening if the user clicks and unclicks a certain box? like if many users are likely to select and then deselect a particular option?